src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -20,24 +20,23 @@ namespace FancyZonesEditor
     public partial class MainWindow : MetroWindow
     {
         public const int MaxZones = 40;
-        private const int PanelItemSize = 150;
-        private readonly Settings _settings = ((App)Application.Current).ZoneSettings[0];
+        public Settings Settings = App.ZoneSettings[MonitorVM.CurrentMonitor];

```

Comment:
If I'm reading this correctly, this is ever initialized once for a specific monitor #, and then never updated if monitor changes. Is this by design (because the MainWindow is created per monitor, perhaps)?

Reply:
@rokhrest if you are talking about the PanelItemSize, it is meant to stay constant indeed. If it is something else can you elaborate?

Reply:
YES! This is something I've been trying to change but it's been tricky. I removed the `readonly` and I created a method to update the monitor settings where I:

```
 _settings = App.ZoneSettings[MonitorVM.CurrentMonitor];
```

BUT... Nothing happens 🤦‍♂️

Anyway, to answer your question, I'm working on changing this!